src/forms/ShowOnYes.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+
+// PROJECT COMPONENTS
+import { ControlledRadioYesNo } from './inputs';
+import { ContentH1 } from '../components/headings';
+
+/**
+ * Yes/No radio buttons. 'Yes' reveals the given Component(s)
+ * 
+ * @param {object} props See below
+ * @param {object} props.clientPartial - Current or future client data.
+ * @param {string} props.propName - Client prop name to store user's choice.
+ * @param {function} props.updateClientValue
+ * @param {string} props.question - Yes/no question for user.
+ * @param {string} props.heading - Heading for this section.
+ * @param {object} props.children - Components to be revealed.
+ * 
+ * @returns {object} Component
+ */
+class ShowOnYes extends React.Component {
+  constructor (props) {
+    super(props);
+    
+    const show = props.clientPartial[ props.propName ];
+    this.state = { showChildren: show };
```

Comment:
There shouldn't be a need to use state here. Just use `client[propName]` directly.